packages/extension/src/shared/token/storage.ts

```diff
@@ -45,3 +45,17 @@ export async function removeToken(token: BaseToken) {
   await assertSchema(baseTokenSchema, token)
   return tokenStore.remove((t) => equalToken(t, token))
 }
+
+
+export async function migrateTokens() {
+  console.debug("Migrating tokens")
+  const allTokens = await tokenStore.get()
+
+  for (const token of allTokens) {
+    if (token.verified === undefined) {
```

Comment:
Do we need to add the `verified` to the schema? https://github.com/alephium/extension-wallet/blob/3724c939558f06839aeafd59fe0550c554af0e9f/packages/extension/src/shared/token/storage.ts#L30

Reply:
Yes, i will add `description` as well. Thanks!